data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report10_extra.py

```diff
@@ -866,26 +866,6 @@ def my_sql(table_name, column_name):
   SELECT person_id
   FROM `{{project_id}}.{{rt_cdr_deid}}.person` o
   )
-  
```

Comment:
@MichaelSchmidt82 you would also need to update the comment for this check to remove primary_consent related  info.